### ISSUE TITLE
Far jumps/returns and IRETs are explicitly lifted to "undefined"

### ISSUE THREAD

**Issue by nshp** [Vector35/binaryninja-api#4031]

**Comment by xusheng6** [comment]
Yeah I thought about this as well. The problem is that these instructions may change the segment registers. However, since the BNIL does not model the segment registers at all for now, it would not be a big loss if we ignore it and emit the IL for ret/jmp. PR is welcome, if you are still interested.
